### ISSUE TITLE
Missing variable crashing quickbrick, preventing generation of truth files.

### ISSUE THREAD

**Issue by weaverba137** [desihub/desisim#62]
A recent run of quickbrick (tag 0.8.2) produced this output:
```
quickbrick -b valid-mix-50 -n 50 --objtype=mix --wavestep=0.2 --downsampling=5 --seed=xxxxx --outdir_truth=yyy                                                                               
INFO:io.py:466:read_basis_templates: Reading /project/projectdirs/desi/spectro/templates/basis_templates/v1.1/star_templates_v1.2.fits                                                                                                                      
/project/projectdirs/desi/software/cori/desisim/0.8.2/lib/python2.7/site-packages/desisim-0.7.dev287-py2.7.egg/desisim/filterfunc.py:68: RuntimeWarning: invalid value encountered in double_scalars                                                        
  flux /= np.trapz(self.interp(wave1)*wave1,wave1)                                                                            
INFO:io.py:466:read_basis_templates: Reading /project/projectdirs/desi/spectro/templates/basis_templates/v1.1/lrg_templates_v1.2.fits                                                                                                                       
INFO:io.py:466:read_basis_templates: Reading /project/projectdirs/desi/spectro/templates/basis_templates/v1.1/qso_templates_v1.1.fits                                                                                                                       
INFO:io.py:466:read_basis_templates: Reading /project/projectdirs/desi/spectro/templates/basis_templates/v1.1/elg_templates_v1.5.fits
INFO:io.py:466:read_basis_templates: Reading /project/projectdirs/desi/spectro/templates/basis_templates/v1.1/star_templates_v1.2.fits
/global/cscratch1/sd/kisner/software/hpcports_gnu/astropy-1.0.1_f4385b62-1.0/lib/python2.7/site-packages/astropy-1.0.1-py2.7-linux-x86_64.egg/astropy/io/fits/fitsrec.py:774: UserWarning: TDIM7 value (10,5) does not fit with the size of the array items (10).  TDIM7 will be ignored.
  actual_nitems, indx + 1))
Traceback (most recent call last):
  File "/project/projectdirs/desi/software/cori/desisim/0.8.2/bin/quickbrick", line 4, in <module>
    __import__('pkg_resources').run_script('desisim==0.7.dev287', 'quickbrick')
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 699, in run_script
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 1617, in run_script
  File "/global/project/projectdirs/desi/software/cori/desisim/0.8.2/lib/python2.7/site-packages/desisim-0.7.dev287-py2.7.egg/EGG-INFO/scripts/quickbrick", line 279, in <module>
    fx.append(fits.ImageHDU(qsim.wavelengthGrid.astype(np.float32), name='_SOURCEWAVE', header=header))
NameError: name 'fx' is not defined
```

**Comment by dkirkby** [comment]
This is a bit off topic but I have noticed that when I get github issue emails, the periods are somehow filtered out of code blocks, resulting in hard to decipher lines like:
```
fxappend(fitsImageHDU(qsimwavelengthGridastype(npfloat32), name='_SOURCEWAVE', header=header))
```
Are others seeing this also?  I am using google inbox for my mail reader.

**Comment by sbailey** [comment]
I also don't see the periods when viewing the reports from gmail on
safari.  Odd.

**Comment by cballand** [comment]
Bug solved in PR #63
